status/status.go

```diff
@@ -16,11 +18,12 @@ const NamePrefix = "Red Hat Trusted App Test"
 // Reporter is a generic interface all status implementations must follow.
 type Reporter interface {
 	ReportStatus(client.Client, context.Context, *tektonv1beta1.PipelineRun) error
+	ReportStatusForSnapshot(client.Client, context.Context, *helpers.IntegrationLogger, *applicationapiv1alpha1.Snapshot, *[]gitops.IntegrationTestStatusDetail) error
```

Comment:
We should use this struct instead of custom types
```suggestion
	ReportStatusForSnapshot(client.Client, context.Context, *helpers.IntegrationLogger, *applicationapiv1alpha1.Snapshot, *SnapshotIntegrationTestStatuses) error
```

Reply:
Thanks. Updated.